test/specs/manifestBuilder.e2e.ts

```diff
@@ -0,0 +1,225 @@
+      workbench,
+      'SFDX: Deploy Source in Manifest to Org',
+      1
+    );
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(
+      workbench,
+      'Running SFDX: Deploy Source to Org',
+      fiveMinutes
+    );
+
+    // Look for the success notification that appears which says, "SFDX: Deploy Source to Org successfully ran".
+    const successNotificationWasFound = await utilities.notificationIsPresent(
+      workbench,
+      'SFDX: Deploy Source to Org successfully ran'
+    );
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify Output tab
+    const outputPanelText = await utilities.attemptToFindOutputPanelText(
+      'Salesforce CLI',
+      'Starting SFDX: Deploy Source to Org',
+      10
+    );
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('=== Deployed Source');
+    expect(outputPanelText).toContain('STATE');
+    expect(outputPanelText).toContain('FULL NAME');
+    expect(outputPanelText).toContain('TYPE');
+    expect(outputPanelText).toContain('PROJECT PATH');
+    expect(outputPanelText).toContain('Customer__c');
+    expect(outputPanelText).toContain('Product__c');
+    expect(outputPanelText).toContain('CustomObject');
+    expect(outputPanelText).toContain(
+      'force-app/main/default/objects/Customer__c/Customer__c.object-meta.xml'
+    );
+    expect(outputPanelText).toContain('ended SFDX: Deploy Source to Org');
+  });
+
+  step('SFDX: Retrieve Source in Manifest from Org', async () => {
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - SFDX: Retrieve Source in Manifest from Org`
+    );
+    // Using the Command palette, run SFDX: Retrieve Source in Manifest from Org
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    await editorView.openEditor('manifest.xml');
+    // Clear output before running the command
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'View: Clear Output',
+      1
+    );
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'SFDX: Retrieve Source in Manifest from Org',
+      1
+    );
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(
+      workbench,
+      'Running SFDX: Retrieve Source from Org',
+      fiveMinutes
+    );
+
+    // Look for the success notification that appears which says, "SFDX: Retrieve Source from Org successfully ran".
+    const successNotificationWasFound = await utilities.notificationIsPresent(
+      workbench,
+      'SFDX: Retrieve Source from Org successfully ran'
+    );
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify Output tab
+    const outputPanelText = await utilities.attemptToFindOutputPanelText(
+      'Salesforce CLI',
+      'Starting SFDX: Retrieve Source from Org',
+      10
+    );
+    expect(outputPanelText).not.toBeUndefined();
```

Comment:
Similar comment as above about being too tightly coupled with the output which we don't control.